internal/service/rekognition/stream_processor.go

```diff
@@ -215,45 +215,42 @@ func (r *resourceStreamProcessor) Schema(ctx context.Context, req resource.Schem
 						objectvalidator.AtLeastOneOf(path.MatchRelative().AtName("bounding_box"), path.MatchRelative().AtName("polygon")),
 					},
 					Blocks: map[string]schema.Block{
-						"bounding_box": schema.SingleNestedBlock{
-							CustomType:  fwtypes.NewObjectTypeOf[boundingBoxModel](ctx),
+						"bounding_box": schema.ListNestedBlock{
+							CustomType:  fwtypes.NewListNestedObjectTypeOf[boundingBoxModel](ctx),
 							Description: "The box representing a region of interest on screen.",
-							Validators: []validator.Object{
-								objectvalidator.AlsoRequires(
-									path.MatchRelative().AtName("height"),
-									path.MatchRelative().AtName("left"),
-									path.MatchRelative().AtName("top"),
-									path.MatchRelative().AtName("width"),
-								),
-								objectvalidator.ConflictsWith(path.MatchRelative().AtParent().AtName("polygon")),
+							Validators: []validator.List{
+								listvalidator.ConflictsWith(path.MatchRelative().AtParent().AtName("polygon")),
+								listvalidator.SizeBetween(1, 1),
 							},
-							Attributes: map[string]schema.Attribute{
-								"height": schema.Float64Attribute{
-									Optional:    true,
-									Description: "Height of the bounding box as a ratio of the overall image height.",
-									Validators: []validator.Float64{
-										float64validator.Between(0.0, 1.0),
+							NestedObject: schema.NestedBlockObject{
+								Attributes: map[string]schema.Attribute{
+									"height": schema.Float64Attribute{
+										Description: "Height of the bounding box as a ratio of the overall image height.",
+										Required:    true,
```

Comment:
Similar to the `polygon` arguments, should these be optional?
https://docs.aws.amazon.com/rekognition/latest/APIReference/API_BoundingBox.html